spec/models/case_contact_spec.rb

```diff
@@ -547,6 +547,19 @@
         end
       end
     end
+
+    describe ".drafts_for_removal" do
+      let!(:active_case_contact) { create(:case_contact, status: "active") }
+      let!(:new_draft_case_contact) { create(:case_contact, status: "started", created_at: Time.now, draft_case_ids: []) }
+      let!(:draft_case_contact_without_draft_case_id) { create(:case_contact, status: "started", created_at: Time.now - 2.days, draft_case_ids: []) }
+      let!(:draft_case_contact_with_draft_case_id) { create(:case_contact, status: "started", created_at: Time.now - 2.days, draft_case_ids: [1]) }
+      let!(:eight_day_draft_case_contact) { create(:case_contact, status: "details", created_at: Time.now - 8.days) }
```

Comment:
can this also be `status: "started"` for consistency?

Reply:
I actually did that intentionally to mimic a draft that was saved correctly after step one (moving it to 'details') but was then abandoned & was over a week old to cover the second half of the ticket (remove any drafts older than a week)